### DISCUSSION TITLE
Decrypt data channel bytes

### DISCUSSION THREAD

**Discussion by HighOnLemonz** [sipsorcery-org/sipsorcery#1250] [Q&A] [answered]
Hey,

Was wondering if the databytes that come on datachannel.onmessage event are already decrypted or is additional decryption required. Just turning the bytes into ascii is not providing any information. Payload protocol is webrtc_binary.

Thank you

**Comment by sipsorcery** [comment, marked as answer]
The underlying SRTP packets with the datachannel payloadd have already been decrypted.

**Comment by HighOnLemonz** [comment]
Thank you
